doc/getting_started/setup_fpga.md

```diff
@@ -180,45 +180,19 @@ To load `hello_world` into the FPGA on the ChipWhisperer CW310 board follow the
    ```console
    screen /dev/ttyACM1 115200,cs8,-ixon,-ixoff
    ```
-1. Run the loading tool.
+1. Run the `opentitantool`.
    ```console
    cd ${REPO_TOP}
-   ./util/fpga/cw310_loader.py --firmware build-bin/sw/device/examples/hello_world/hello_world_fpga_cw310.bin --set-pll-defaults
-   ```
-
-   This should report how the binary is split into frames:
-   ```
-   CW310 Loader: Attemping to find CW310 FPGA Board:
-       No bitstream specified
-   Board found, setting PLL2 to 100 MHz
-   INFO: Programming firmware file: build-bin/sw/device/examples/hello_world/hello_world_fpga_cw310.bin
-   Programming OpenTitan with "build-bin/sw/device/examples/hello_world/hello_world_fpga_cw310.bin"...
-   Transferring frame 0x00000000 @ 0x00000000.
-   Transferring frame 0x00000001 @ 0x000007D8.
-   Transferring frame 0x00000002 @ 0x00000FB0.
-   Transferring frame 0x00000003 @ 0x00001788.
-   Transferring frame 0x00000004 @ 0x00001F60.
-   Transferring frame 0x00000005 @ 0x00002738.
-   Transferring frame 0x00000006 @ 0x00002F10.
-   Transferring frame 0x80000007 @ 0x000036E8.
-   Loading done.
+   bazel run //sw/host/opentitantool set-pll # This needs to be done only once.
+   bazel run //sw/host/opentitantool bootstrap build-bin/sw/device/examples/hello_world/hello_world_fpga_cw310.bin
```

Comment:
```suggestion
   bazel run //sw/host/opentitantool bootstrap $($REPO/ci/scripts/target-location.sh //sw/device/examples/hello_world:hello_world_fpga_cw310_bin)
```

Reply:
The paths should be absolute since bazel runs `opentitantool` somewhere else. I made a small change in `ci/scripts/target-location.sh`, PTAL.